src/common/config_opts.h

```diff
@@ -1164,7 +1164,8 @@ OPTION(kstore_onode_map_size, OPT_U64, 1024)
 OPTION(kstore_cache_tails, OPT_BOOL, true)
 OPTION(kstore_default_stripe_size, OPT_INT, 65536)
 
-OPTION(filestore_omap_backend, OPT_STR, "leveldb")
+/// change default from leveldb to rocksdb
```

Comment:
don't need this comment; source file reflects how things are, not how they got that way